### DISCUSSION TITLE
Train model on custom dataset

### DISCUSSION THREAD

**Discussion by kachhadiyaraj15** [obss/sahi#899] [Q&A]
Hello everyone, 
I want to train the model on my custom dataset. Please anyone help me with training on my dataset?

**Comment by kachhadiyaraj15** [comment]
I think, code of import model sahi.utils.yolov8, it is actual model of yolov8 
right? 
So I can train yolov8 on my custom dataset and then process further?

**Reply by drdoof2019** [reply]
Yes
